.env.example

```diff
@@ -10,3 +10,4 @@ FIREBASE_PRIVATE_KEY=""
 FIREBASE_PROJECT_ID=""
 FIREBASE_CLIENT_EMAIL=""
 FIREBASE_DATABASE_URL=""
+GITHUB_TOKEN=""
```

Comment:
My concern here, it will create it as an App not the user. Is this an issue? 🤔

Reply:
By "create it as an App" do you mean actually creating the issue? If so, that's not the purpose of having the github token. The token is for the request to be authenticated and have a more flexible rate limit (30 requests per minute instead of 10)

Reply:
The bot would create the issue for the user. Eddie bot will get all the green squares :green_square:

Reply:
Ah ok, this doesn't create an issue. Otherwise yes the bot will get all the green squares :D